script.js

```diff
@@ -0,0 +1,432 @@
+      ['Ctrl', 'Win', 'Alt', 'Space', 'Alt', 'Left', 'Down', 'Right', 'Ctrl'],
+    ];
+
+    const rusLayout = [
+      ['ё', '1', '2', '3', '4', '5', '6', '7', '8', '9', '0', '-', '=', wideButtons[0]],
+      ['Tab', 'й', 'ц', 'у', 'к', 'е', 'н', 'г', 'ш', 'щ', 'з', 'х', 'ъ', '\\', 'DEL'],
+      [wideButtons[1], 'ф', 'ы', 'в', 'а', 'п', 'р', 'о', 'л', 'д', 'ж', 'э', wideButtons[2]],
+      [wideButtons[3], 'я', 'ч', 'с', 'м', 'и', 'т', 'ь', 'б', 'ю', '.', 'Up', wideButtons[3]],
+      ['Ctrl', 'Win', 'Alt', 'Space', 'Alt', 'Left', 'Down', 'Right', 'Ctrl'],
+    ];
+
+    const keyLayout = engLayout;
+
+    // Create keys
+    for (let i = 0; i < keyLayout.length; i += 1) {
+      for (let j = 0; j < keyLayout[i].length; j += 1) {
+        const key = document.createElement('button');
+        key.setAttribute('type', 'button');
+        key.classList.add('keyboard__key');
+        key.cl = codeLayout[i][j];
+        key.eng = engLayout[i][j];
+        key.rus = rusLayout[i][j];
+        key.seng = ENGshiftLayout[i][j];
+        key.srus = RUSshiftLayout[i][j];
+        if (wideButtons.includes(keyLayout[i][j])) {
+          key.classList.add('keyboard__key--wide');
+        }
+        if (keyLayout[i][j] === 'Space') {
+          key.classList.add('keyboard__key--space');
+        }
+        if (keyLayout[i][j] === 'CapsLock') {
+          key.classList.add('keyboard__key--activatable');
+          if (keyboard.shift) {
+            key.classList.add('keyboard__key--active-on');
+          }
+        }
+
+        key.innerHTML = `<span class="ENG keySpan">${engLayout[i][j]}</span><span class="RUS keySpan">${rusLayout[i][j]}</span><span class="SHIFT keySpan">${ENGshiftLayout[i][j]}</span><span class="RUS_SHIFT keySpan">${RUSshiftLayout[i][j]}</span>`;
+
+        if (keyboard.lang === 'en' && keyboard.shift === false) {
+          key.querySelector('.RUS').classList.add('HIDDEN');
+          key.querySelector('.SHIFT').classList.add('HIDDEN');
+          key.querySelector('.RUS_SHIFT').classList.add('HIDDEN');
+        } else if (keyboard.lang === 'en' && keyboard.shift === true) {
+          key.querySelector('.RUS').classList.add('HIDDEN');
+          key.querySelector('.ENG').classList.add('HIDDEN');
+          key.querySelector('.RUS_SHIFT').classList.add('HIDDEN');
+        } else if (keyboard.lang === 'rus' && keyboard.shift === false) {
+          key.querySelector('.ENG').classList.add('HIDDEN');
+          key.querySelector('.SHIFT').classList.add('HIDDEN');
+          key.querySelector('.RUS_SHIFT').classList.add('HIDDEN');
+        } else if (keyboard.lang === 'rus' && keyboard.shift === true) {
+          key.querySelector('.RUS').classList.add('HIDDEN');
+          key.querySelector('.SHIFT').classList.add('HIDDEN');
+          key.querySelector('.ENG').classList.add('HIDDEN');
+        }
+
+        key.classList.add(`keyboard__key-code-${codeLayout[i][j]}`);
+
+        key.addEventListener('click', () => {
```

Comment:
I added an event listener to each button, because it was shown in the video tutorial. AND later on I larned about event delegation, but it was quite too late for deeply refactoring the whole code, so that's why I did nothing about that. But now I know that lots of event listeners seriously lower application performance and I'll never do that in future...